lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -332,8 +332,21 @@ module.exports = class MethodMappings {
         async switchToFrame(frameId) {
           if (frameId === undefined) {
             frameId = null;
+          } else if (typeof frameId === 'string'){
+            try {
+              frameId = await this.driver.findElement(By.id(frameId));
+            } catch (err){
+              try {
```

Comment:
Rather than putting a try block inside a catch, I prefer if the try blocks are nested within other try blocks and the catch statements follow.